worlds/TheLocalPub/LamentinLosers

```diff
@@ -0,0 +1 @@
+
```

Comment:
remove this file and create a folder called LamentinLosers within TheLocalPub to contain the mission

Reply:
What? there is..